### PR TITLE
openapi spec for GET importcandidate endpoint created

### PR DESCRIPTION
**Pull request by truhacevkir**

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -488,6 +488,49 @@ paths:
...
+            application/ld+json:
+              schema:
+                $ref: '#/components/schemas/Publication'
+            application/vnd.datacite.datacite+xml:
+              schema:
+                $ref: '#/components/schemas/PublicationDataCiteXml'
+            application/vnd.schemaorg.ld+json:
+              schema:
+                $ref: '#/components/schemas/PublicationSchemaOrg'
+        '410':
```
Comment (anetteOlli) [inline docs/openapi.yaml:527]: Not found?

File: docs/openapi.yaml [hunk 2]
```diff
@@ -488,6 +488,49 @@ paths:
...
+          description: Permanently gone
```
Comment (anetteOlli) [inline docs/openapi.yaml:528]: I don't think we will logically delete stuff, only physically delete candidates.

Resource service uses this response if a publication has status = deleted.

An import candidate doesn't have that status field.

File: docs/openapi.yaml [hunk 3]
```diff
@@ -612,6 +655,134 @@ components:
             type: string
         title:
           type: string
+    ImportCandidate:
```
Comment (anetteOlli) [inline docs/openapi.yaml:658]: Is it possible to extend ImportCandidate from publication like you do in typescript in open-api? Just wondering

File: docs/openapi.yaml [hunk 4]
```diff
@@ -488,6 +488,49 @@ paths:
...
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Publication'
+            application/ld+json:
+              schema:
+                $ref: '#/components/schemas/Publication'
+            application/vnd.datacite.datacite+xml:
+              schema:
+                $ref: '#/components/schemas/PublicationDataCiteXml'
```
Comment (anetteOlli) [inline docs/openapi.yaml:523]: We are not going to return DataCiteXml from this endpoint.

File: docs/openapi.yaml [hunk 5]
```diff
@@ -488,6 +488,49 @@ paths:
...
+            application/vnd.schemaorg.ld+json:
+              schema:
+                $ref: '#/components/schemas/PublicationSchemaOrg'
```
Comment (anetteOlli) [inline docs/openapi.yaml:526]: Ask Rurik if we need to return this content-type

**Review by anetteOlli — CHANGES_REQUESTED** [review]

File: docs/openapi.yaml (hunk 3 above)
Comment (truhacevkir) [reply docs/openapi.yaml:658]: i will test "allOf" property in openapi scheme, changing it back if it does not work

File: docs/openapi.yaml (hunk 5 above)
Comment (truhacevkir) [reply docs/openapi.yaml:526]: trenger ikke nei

File: docs/openapi.yaml [hunk 6]
```diff
@@ -488,6 +488,46 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+
+  /import-candidate/{identifier}:
+    get:
+      summary: Fetch Import candidate by Identifier
+      tags:
+        - external
```
Comment (sondrev) [inline docs/openapi.yaml:496]: You dont need this tag unless you want the call to be documented for externals. 
I also see you  dont have the "secuirty" property. Do you want this to be publicly accessible?

**Pull request closed without merging**